### ISSUE TITLE
Implement backend handler for adding workshop to database

### ISSUE THREAD

**Issue by rupsis** [rustbridge/rustbridge.io#41]
- [ ] create workshop table

- [ ] implement post method for populating table

**Comment by technetos** [comment]
@rupsis can you clarify what this issue is?

**Comment by rupsis** [comment]
Yeah, this is really 2 issues. We need to design the database schema for workshops, and create/implement the back end logic to actually create the workshop. During the hackathon, we had created a very basic workshop submission form, but hadn't implemented it yet.
